shell/platform/darwin/macos/framework/Source/FlutterPlatformViews.h

```diff
@@ -0,0 +1,62 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_FLUTTERPLATFORMVIEWS_H_
+#define FLUTTER_FLUTTERPLATFORMVIEWS_H_
+
+#import <AppKit/AppKit.h>
+
+#import "FlutterCodecs.h"
+#import "FlutterMacros.h"
+
+NS_ASSUME_NONNULL_BEGIN
```

Comment:
please remove all uses of `NS_ASSUME_NONNULL_BEGIN` from the PR and replace with individual nullability annotations.